src/main/java/net/sf/jabref/gui/importer/fetcher/ISBNtoBibTeXFetcher.java

```diff
@@ -101,7 +102,8 @@ public boolean processQuery(String query, ImportInspector inspector, OutputPrint
                 bibtexString = scan.useDelimiter("\\A").next();
             }
 
-            BibEntry entry = BibtexParser.singleFromString(bibtexString);
```

Comment:
When that happens, it is easy to remove it from the  `ImportFormatPreferences`. At the moment it is the most feasible way to get rid of `Globals` dependencies.